common/src/main/scala/org/apache/celeborn/common/metrics/MetricLabels.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.common.metrics
+
+private[metrics] trait MetricLabels {
+  val labels: Map[String, String]
+
+  def labelString: String = MetricLabels.labelString(labels)
+}
+
+object MetricLabels {
+  def labelString(labels: Map[String, String]): String = {
```

Comment:
This method should ensure that if two maps contain the same entries, the return value should be identical. However HashMaps don't guarantee order. I think we can simply sort the strings before concat together(under the assumption that the labels will not be too many)
```
labels.map { case (key: String, value: String) => s"""$key="$value"""" }.toList.sorted.mkString
```